AdobeStockImageAdminUi/Test/Mftf/Test/AdminAdobeStockSavedLicensedImageLocateTest.xml

```diff
@@ -0,0 +1,61 @@
+            <group value="adobe_stock_integration"/>
+        </annotations>
+        <before>
+            <actionGroup ref="LoginAsAdmin" stepKey="loginAsAdmin"/>
+            <actionGroup ref="AdminOpenMediaGalleryForPageNoEditorActionGroup" stepKey="openMediaGalleryForPage"/>
+            <actionGroup ref="AdminAdobeStockOpenPanelFromMediaGalleryActionGroup" stepKey="openAdobeStockPanel"/>
+            <actionGroup ref="AdminAdobeStockClickSignInActionGroup" stepKey="clickOnSignIn"/>
+            <actionGroup ref="AdminAdobeStockImsPopupSignInFillUserDataActionGroup" stepKey="fillUserCredentials"/>
+            <actionGroup ref="AdminAdobeStockImsPopupClickSignInActionGroup" stepKey="clickSignInImsPopup"/>
+            <actionGroup ref="AdminAdobeStockAssertUserLoggedActionGroup" stepKey="assertUserLoggedIn"/>
+            <actionGroup ref="AdminSearchImagesOnModalActionGroup" stepKey="searchForLicensedImage">
+                <argument name="query" value="{{AdobeStockConfigDataLicensedImage.value}}"/>
+            </actionGroup>
+            <actionGroup ref="AdminAdobeStockExpandImagePreviewActionGroup" stepKey="expandLicensedImage"/>
+            <click selector="{{AdobeStockImagePreviewSection.saveLicensedImage}}" stepKey="clickSave"/>
+            <waitForPageLoad stepKey="waitForPromptModal"/>
+            <grabValueFrom selector="{{AdobeStockImagePreviewSection.generatedImageName}}" stepKey="grabSaveImageFileName"/>
+            <click selector="{{AdobeStockImagePreviewSection.confirm}}" stepKey="clickOnPopupConfirm"/>
+            <waitForPageLoad stepKey="waitForMediaGalleryOpen"/>
+            <seeElement selector="{{AdobeStockSection.mediaGalleryImage({$grabSaveImageFileName})}}" stepKey="assertSavedImage"/>
+            <actionGroup ref="AdminAdobeStockOpenPanelFromMediaGalleryActionGroup" stepKey="openAdobeStockPanelAgain"/>
+        </before>
+        <after>
+            <actionGroup ref="AdminOpenMediaGalleryForPageNoEditorActionGroup" stepKey="openMediaGalleryForPage"/>
+            <actionGroup ref="AdminAdobeStockOpenPanelFromMediaGalleryActionGroup" stepKey="openAdobeStockPanel"/>
+            <actionGroup ref="AdminAdobeStockUserSignOutActionGroup" stepKey="adobeLogout"/>
+            <actionGroup ref="AdminAdobeStockAssertUserNotLoggedActionGroup" stepKey="assertAdobeUserLoggedOut"/>
+            <actionGroup ref="resetAdminDataGridToDefaultView" stepKey="resetAdminDataGridToDefaultView"/>
+            <actionGroup ref="logout" stepKey="adminLogout"/>
+        </after>
+        <actionGroup ref="AdminSearchImagesOnModalActionGroup" stepKey="searchForLicensedImage">
+            <argument name="query" value="{{AdobeStockConfigDataLicensedImage.value}}"/>
+        </actionGroup>
+        <actionGroup ref="AdminAdobeStockExpandImagePreviewActionGroup" stepKey="expandLicensedImage"/>
+        <grabAttributeFrom selector="{{AdobeStockImagePreviewSection.locateImage}}" userInput="name" stepKey="grabImageFileName"/>
```

Comment:
This line returns the empty string in my local test r un, as the span element containing the Locate text does not have a `name` attribute. This results in the rest of the test being unable to compare against the file name. Here's a snippet of this happening in my test run locally (note the empty string used with the `alt` attribute):

```
[searchForLicensedImage] AdminSearchImagesOnModalActionGroup
  [fillQueryInput] fill secret field "input.data-grid-search-control","84Gol3inT8K17CaG7uHIhg=="
  [applySearch] click "#words + button.action-submit"
  [waitForSearchForQuery] wait for loading mask to disappear
[expandLicensedImage] AdminAdobeStockExpandImagePreviewActionGroup
  [clickOnThumbnail] click "aside.adobe-stock-modal [data-role=grid-wrapper] img[data-role=thumbnail]"
  [waitForImagePreviewToExpand] wait for loading mask to disappear
[grabImageFileName] grab attribute from "//button[@class='action-secondary']//span[text()='Locate']","name"
[clickLocate] click "//button[@class='action-secondary']//span[text()='Locate']"
[waitForMediaGalleryOpen] wait for page load 10
[assertSavedImage] see element "//img[contains(@alt,'')]"
[deleteImageActionGroup] AdminMediaGalleryDeleteImage
  [seeImageToDelete] see element "//img[contains(@alt,'')]"
  [clickImageToDelete] click "//img[contains(@alt,'')]"
[saveScreenshot] save screenshot
```

Reply:
Hey @filmaj, I just tested it locally and can confirm that this test just passed and the `name` attribute with the filename is present in the _Locate_ button. :thinking: Could you check if actions.js file in this PR is loaded in your browser with the method getFileName()?

Reply:
As you pointed  out to me on Slack, I must  not be in developer  mode and client side assets a re being cached in my system. After doing  that song and dance, I get a different error around finding  the image in the media gallery once clicking the Locate button:

```
1) AdminAdobeStockSavedLicensedImageLocateTestCest: Admin adobe stock saved licensed image locate test
 Test  tests/functional/Magento/FunctionalTest/_generated/adobe_stock_integration_suite/AdminAdobeStockSavedLicensedImageLocateTestCest.php:AdminAdobeStockSavedLicensedImageLocateTest
 Step  See element "//img[contains(@alt,'hygge-concept-with-cat-book-and.jpeg')]"
 Fail  Failed asserting that an array is not empty.
```

I think it may be an issue with my environment. When clicking the Locate button, it takes a long time for the media gallery to load up on my local instance...